app/controllers/concerns/secured.rb

```diff
@@ -12,7 +12,8 @@ module Secured
 
   def authenticate_request!
     auth_token
-  rescue JWT::VerificationError, JWT::DecodeError
+  rescue JWT::VerificationError, JWT::DecodeError => e
+    logger.info { e.message }
```

Comment:
Thanks kasirajr for this another PR
Just a small update, I was wondering about give a more friendly message, something like
`logger.info "Authentication failed due to: #{e.message}"`

Could you update?

Reply:
updated the message